ansible/playbooks/roles/repository/files/download-requirements/src/config/config.py

```diff
@@ -207,17 +213,18 @@ def __add_args(self, argv: List[str]):
         self.distro_subdir = self.family_subdir / self.os_type.os_name
 
         # add optional arguments
+        self.dest_manifest = args['manifest'] or None
+        self.full_download = args['full_download']
+        self.is_log_file_enabled = False if args['no_logfile'] else True
         self.repos_backup_file = Path(args['repos_backup_file'])
         self.retries = args['retries']
-        self.is_log_file_enabled = False if args['no_logfile'] else True
-        self.dest_manifest = args['manifest'] or None
         self.verbose_mode = True if self.log_level == logging.DEBUG else args['verbose']
 
         # offline mode
         self.rerun = args['rerun']
         self.pyyaml_installed = args['pyyaml_installed']
 
-    def __print_parsed_manifest_data(self, requirements: Dict[str, Any], manifest: Dict[str, Any]):
+    def __get_parsed_manifest_data_output(self, requirements: Dict[str, Any], manifest: Dict[str, Any]) -> str:
```

Comment:
SonarCloud reports that `requirements` parameter is unused, how about removing it?